src_server/Api.re

```diff
@@ -0,0 +1,34 @@
+open Express;
+
+let dirname =
+  switch ([%bs.node __dirname]) {
+  | Some(d) => d
+  | None => Js.Exn.raiseError("Could not get __dirname")
+  };
+
+let getRefdomainsJson = () => {
+  Node.Fs.readFileAsUtf8Sync(Node.Path.join([|dirname, "Refdomains.json"|]))
+  |> Js.Json.parseExn;
+};
+
+let app = express();
+
+App.disable(app, ~name="x-powered-by");
+
+App.get(app, ~path="/refdomains") @@
+Middleware.from((next, req) =>
+  switch (Request.baseUrl(req)) {
+  | "" => Response.sendJson(getRefdomainsJson())
+  | _ => next(Next.route)
+  }
+);
+
+let onListen = e =>
```

Comment:
Nitpick: use different variable names for event and error just to avoid possible confusion.